diff -Naur modulemd-2.8.0/meson.build modulemd-2.8.0.tpg/meson.build
--- modulemd-2.8.0/meson.build	2019-09-04 01:35:13.000000000 +0000
+++ modulemd-2.8.0.tpg/meson.build	2019-09-25 17:45:37.546319377 +0000
@@ -52,7 +52,6 @@
 pkg = import('pkgconfig')
 gobject = dependency('gobject-2.0')
 yaml = dependency('yaml-0.1')
-gtkdoc = dependency('gtk-doc')
 
 with_rpmio = get_option('rpmio')
 with_libmagic = get_option('libmagic')
@@ -61,22 +60,11 @@
 magic = cc.find_library('magic', required : with_libmagic)
 
 glib_prefix = dependency('glib-2.0').get_pkgconfig_variable('prefix')
-glib_docpath = join_paths(glib_prefix, 'share', 'gtk-doc', 'html')
 
 sh = find_program('sh')
 sed = find_program('sed')
 test = find_program('test')
 
-ret = run_command ([test, '-e', join_paths(glib_docpath, 'glib/index.html')])
-if ret.returncode() != 0
-  error('Missing documentation for GLib.')
-endif
-
-ret = run_command ([test, '-e', join_paths(glib_docpath, 'gobject/index.html')])
-if ret.returncode() != 0
-  error('Missing documentation for GObject.')
-endif
-
 python_name = get_option('python_name')
 
 if python_name != ''
diff -Naur modulemd-2.8.0/modulemd/meson.build modulemd-2.8.0.tpg/modulemd/meson.build
--- modulemd-2.8.0/modulemd/meson.build	2019-09-04 01:35:13.000000000 +0000
+++ modulemd-2.8.0.tpg/modulemd/meson.build	2019-09-25 17:45:16.735389078 +0000
@@ -479,20 +479,3 @@
   output : 'version.xml',
   configuration : xcdata
 )
-
-gnome.gtkdoc(
-    'modulemd-2.0',
-    install_dir: 'modulemd-2.0',
-    src_dir : './modulemd',
-    main_xml : 'modulemd-docs.xml',
-    gobject_typesfile : join_paths(meson.current_build_dir(), 'modulemd-2.0.types'),
-    dependencies : [
-        modulemd_dep,
-    ],
-    fixxref_args: [
-                 '--extra-dir=@0@'.format(join_paths(glib_docpath, 'glib')),
-                 '--extra-dir=@0@'.format(join_paths(glib_docpath, 'gobject')),
-               ],
-    install : true,
-)
-
